pkg/operator/ceph/cluster/mon/spec.go

```diff
@@ -241,7 +233,7 @@ func (c *Cluster) makeMonDaemonContainer(monConfig *monConfig) v1.Container {
 			[]string{
 				"--foreground",
 				"--public-addr", joinHostPort(monConfig.PublicIP, monConfig.Port),
-				// --public-bind-addr is set in the config file at init time
```

Comment:
Is this the primary change thus far? It looks like a lot of other code moved around, but hard to tell if it changed. Perhaps the code that moved around could be in a separate commit from this core change?

Reply:
I agree. I'll split these changes.